src/firefly/java/edu/caltech/ipac/firefly/server/visualize/VisServerOps.java

```diff
@@ -1434,9 +1437,66 @@ public static WebPlotResult getDS9Region(String fileKey) {
             retval = createError("on getDSRegion", null, e);
         }
         return retval;
+    }
 
+    private static final Map<String, String> footprintMap;
+    static {
```

Comment:
besides the issue regrading where to put the footprint files, I also have some concern regrading if it is ok to hardcode the footprint map in the java file? or put the map data into a text file and create the map by loading the text file?